packages/pool/contracts/DelegationUtils.sol

```diff
@@ -6,9 +6,12 @@ import "./interfaces/IDelegationUtils.sol";
 
 /// @title Contract that implements voting power delegation
 abstract contract DelegationUtils is RewardUtils, IDelegationUtils {
+
+    string internal constant ERROR_DELEGATION_BALANCE = "User that is delegating vote should have enough voting power to at least create a voting proposal";
```

Comment:
This shouldn't have passed linting due to length

Reply:
Fixed